libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -213,6 +252,20 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
     this._destroy$.complete();
   }
 
+  /** @internal gets the dimensions of the component and renders */
+  _updateDimensions(): void {
+    if (this._platform.isBrowser) {
+      this._width = Math.min(
+        MAX_WIDTH,
+        Math.max(
+          MIN_WIDTH,
+          this._elementRef.nativeElement.getBoundingClientRect().width,
+        ),
+      );
```

Comment:
Please do not clamp the width of the svg, the container should take care of the sizing. Please revert to the original behavior.